cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/PropertyUtils.java

```diff
@@ -17,7 +17,6 @@
  */
```

Comment:
Difff is a bit complex in this file... Is there any actual change? Or only indent and `@author` changes?

Reply:
It only changes indent and @author.

Reply:
Ok. NTC